payments-core/src/main/java/com/stripe/android/payments/core/analytics/ErrorReporter.kt

```diff
@@ -50,6 +50,25 @@ interface ErrorReporter {
                 .build()
                 .errorReporter
         }
+
+        fun getAdditionalParamsFromError(error: Throwable): Map<String, String> {
+            return when (error) {
+                is StripeException -> getAdditionalParamsFromStripeException(error)
+                else -> getAdditionalParamsFromStripeException(StripeException.create(error))
+            }
+        }
+        fun getAdditionalParamsFromStripeException(stripeException: StripeException): Map<String, String> {
```

Comment:
I'm surprised the linter doesn't require an extra line between these functions.

Reply:
hmm it might. I added an extra line. I forgot to lint after fixing my merge conflicts and I don't think it had run via CI when you reviewed